SamplesDashboard/ClientApp/src/store/WeatherForecasts.ts

```diff
@@ -0,0 +1,91 @@
+import { Action, Reducer } from 'redux';
```

Comment:
I don't think you need this file anymore

Reply:
Noted. I have deleted the unnecessary files